tests/testthat/helper-funs.R

```diff
@@ -0,0 +1,35 @@
+basicScatP <- function() {
+  ggplot2::ggplot(data = mtcars) +
+    ggplot2::geom_point(ggplot2::aes(x = wt, y = drat)) 
+}
+
+ggMarg2 <- function(type, ...) 
+  ggMarginal(p = basicScatP(), type = type, ...)
+
+funList <- 
+  list(
```

Comment:
So this list wouldn't have to change in order to accommodate testing certain plots only in some versions?